lib/workers/pr/index.ts

```diff
@@ -33,6 +33,15 @@ async function addCodeOwners(
   return [...new Set(assigneesOrReviewers.concat(await codeOwnersForPr(pr)))];
 }
 
+function filterUnavailableUsers(
+  config: RenovateConfig,
+  users: string[]
+): Promise<string[]> {
+  return config.filterOutUnavailableUsers && platform.filterUnavailableUsers
```

Comment:
Log a warning if platform doesn't supports this and user configured it.
Should be done once in repo config init.

Reply:
Makes sense! Done with e1edc49687f0715a470a2bdaf3d5a3ec7fe99d04